docker-compose.yaml

```diff
@@ -8,3 +8,9 @@ services:
     environment:
       APPINSIGHTS_CLOUDROLE: fcp-fd-comms-local
       APPINSIGHTS_CONNECTIONSTRING: ${APPINSIGHTS_CONNECTIONSTRING}
+      MESSAGE_QUEUE_HOST: ${MESSAGE_QUEUE_HOST}
+      MESSAGE_QUEUE_USER: ${MESSAGE_QUEUE_USER}
+      MESSAGE_QUEUE_PASSWORD: ${MESSAGE_QUEUE_PASSWORD}
+      MESSAGE_QUEUE_SUFFIX: fcp-fd-comms-${DEV_SUFFIX}
```

Comment:
We need to keep the topic name and subscription names. They might not necessarily be the same.

Reply:
Have put them back in.